### PR TITLE
Use a value specified by <set-value> element

### PR DESCRIPTION
**Pull request by msrubar**
This is a fix for an issue which was reported by Jacob Varughese.

See: https://fedorahosted.org/openscap/ticket/498

### CODE REVIEW

**Comment by isimluk** [comment]
The trac ticket has an exemplary content attached. Can you please re-use that content to create a test case?

File: src/SCE/sce_engine.c [hunk 1]
```diff
@@ -372,7 +372,11 @@ xccdf_test_result_type_t sce_engine_eval_rule(struct xccdf_policy *policy, const
 
 		char* name = xccdf_value_binding_get_name(binding);
 		xccdf_value_type_t type = xccdf_value_binding_get_type(binding);
-		char* value = xccdf_value_binding_get_value(binding);
+		char* value = xccdf_value_binding_get_setvalue(binding);
+		if (value == NULL)
+		{
+				value = xccdf_value_binding_get_value(binding);
```
Comment (mpreisler) [inline src/SCE/sce_engine.c]: 2 tab indent?

**Comment by msrubar** [comment]
To **isimluk**: thank you for the tips. I got inspired in the ```tests/sce/test_sce.sh``` script. Anyway it should be all fine now.

**Comment by isimluk** [comment]
Perfect.

Thank You!

**Comment by mpreisler** [comment]
One question, doesn't this change break the refine-value functionality with SCE? What if there is no `set-value` but `refine-value` instead?

**Pull request merged**